UPGRADING

```diff
@@ -76,6 +76,10 @@ PHP 8.3 UPGRADE NOTES
 - Dom:
   . Changed DOMCharacterData::appendData() tentative return type to true.
 
+- Intl:
+  . IntlDateformatter::setTimeZone (and its alias dafmt_set_timezone) 
```

Comment:
FYI: it's actually the other way around. `dafmt_set_timezone` has the implementation, and `IntlDateformatter::setTimeZone` is the alias. `@alias foo` is stubs basically mean that "this function/method is an alias of `foo`"

Reply:
good point